rosdep/python.yaml

```diff
@@ -4294,31 +4294,7 @@ python-sip:
   slackware:
     slackpkg:
       packages: [sip]
-  ubuntu:
-    artful: [python-sip-dev]
-    artful_python3: [python3-sip-dev]
-    bionic: [python-sip-dev]
-    bionic_python3: [python3-sip-dev]
```

Comment:
Please leave the existing `_python3` rules so that we don't break people who might be using them. Our general policy is to let them age out when the platform goes EOL. 

We can get the same forwards compatibilty by using a `'*': [python-sip-dev]` rule next to just the legacy python3 rules while not breaking those using the deprecated style.